apps/o2-blocks/README.md

```diff
@@ -26,3 +26,11 @@ Blocks are registered by providing a `name` and `settings` like this:
 ```js
 registerBlockType( 'prefix/name', { /* settings */ } );
 ```
+
+## Building
+
+To build bundle in `apps/o2-blocks/dist`, run:
+
+```bash
+npx lerna run build --scope="*/o2-blocks
```

Comment:
Lerna's `--scope` and `--ignore` match package names, might be good to make that clear:

```suggestion
npx lerna run build --scope="@automattic/o2-blocks
```

Reply:
+1 to use the package name on the docs.

Note that the closing quotes are missing.

```suggestion
npx lerna run build --scope="@automattic/o2-blocks"
```